CHANGES.rst

```diff
@@ -2,11 +2,14 @@
 Changelog
 =========
 
-Plateau 4.2.3 (2023-11-??)
+Plateau 4.3.0 (unreleased)
```

Comment:
Do you want to release today or next year?

Reply:
I'm happy to do a release today if you're OK with that.